string frame_id
string object_type
float64 max_range
---
bool result
string object_type
string sub_type
geometry_msgs/Point position
